<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindowClass</class>
 <widget class="QDialog" name="SettingsWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>730</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>690</width>
    <height>730</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="camera1_groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>591</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera 1</string>
   </property>
   <widget class="QLabel" name="cam1_label_Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Type">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_FPS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Resolution">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolution:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Width">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Height">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Connection">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Color">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam1_lineEdit_Name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNoEditMenu|Qt::ImhNoPredictiveText|Qt::ImhPreferLatin</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam1_lineEdit_FPS">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam1_lineEdit_Width">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam1_lineEdit_Height">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam1_lineEdit_Connection">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cam1_Type_comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cam1_checkBox_Color">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cam1_saveChanges_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="camera1_groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>591</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera 2</string>
   </property>
   <widget class="QLabel" name="cam1_label_Name_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Type_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_FPS_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Resolution_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolution:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Width_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Height_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Connection_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Color_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam2_lineEdit_Name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNoEditMenu|Qt::ImhNoPredictiveText|Qt::ImhPreferLatin</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam2_lineEdit_FPS">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam2_lineEdit_Width">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam2_lineEdit_Height">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam2_lineEdit_Connection">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cam2_Type_comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cam2_checkBox_Color">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cam2_saveChanges_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="camera1_groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>591</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera 3</string>
   </property>
   <widget class="QLabel" name="cam1_label_Name_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Type_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_FPS_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Resolution_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolution:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Width_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Height_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Connection_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Color_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam3_lineEdit_Name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNoEditMenu|Qt::ImhNoPredictiveText|Qt::ImhPreferLatin</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam3_lineEdit_FPS">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam3_lineEdit_Width">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam3_lineEdit_Height">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam3_lineEdit_Connection">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cam3_Type_comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cam3_checkBox_Color">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cam3_saveChanges_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="camera1_groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>591</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera 4</string>
   </property>
   <widget class="QLabel" name="cam1_label_Name_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Type_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_FPS_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Resolution_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolution:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Width_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Height_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Connection_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam1_label_Color_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam4_lineEdit_Name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNoEditMenu|Qt::ImhNoPredictiveText|Qt::ImhPreferLatin</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam4_lineEdit_FPS">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam4_lineEdit_Width">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam4_lineEdit_Height">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cam4_lineEdit_Connection">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cam4_Type_comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cam4_checkBox_Color">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cam4_saveChanges_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
